views/includes/footer.ejs

```diff
@@ -1,14 +1,10 @@
 <footer>
   <div class="f-info">
-    <div class="f-info-socials">
-      <i class="fa-brands fa-square-facebook"></i>
-      <i class="fa-brands fa-square-instagram"></i>
-      <i class="fa-brands fa-linkedin"></i>
+    <div class="fw-bold mt-3">
+      Made with ❤️ by Sanam
     </div>
-    <div class="f-info-brand">&copy; Wanderlust Private Limited</div>
-    <div class="f-info-links">
-      <a href="/privacy">Privacy</a>
-      <a href="/terms">Terms</a>
+    <a class="f-info-brand nav-link" href="/listings"> &copy; Wanderlust Private Limited</a>
+      <div>Facebook • Instagram • Twitter</div>
```

Comment:
if you have added this that means there must be some links. Don't just add this for writing it.